### DISCUSSION TITLE
Is there a way to prefer overflow rather than flip?

### DISCUSSION THREAD

**Discussion by zhaoyao91** [floating-ui/floating-ui#1310] [Q&A] [answered]
Currently, I both use flip and prevent overflow. For example, placement is top, and flip fallback placements are [top, bottom, right, left].

There is room for tooltip in the top, but the tooltip will be pulled back a little to prevent overflow. But if flip is used, it will just flip without trying to prevent overflow.

what I need is like this

![image](https://user-images.githubusercontent.com/3808838/118111872-dcb40e80-b416-11eb-9790-e277817f4362.png)

but it actually behavors like this

![image](https://user-images.githubusercontent.com/3808838/118111912-e76ea380-b416-11eb-9416-1220783b3875.png)

**Comment by atomiks** [comment, marked as answer]
There is, but for some reason it's not documented

These boolean options exist for `flip`:

```
mainAxis,
altAxis
```

I think you want to set `altAxis` to `false`

**Reply by zhaoyao91** [reply]
@atomiks thanks, it works very well :)
